composed.go

```diff
@@ -44,6 +44,14 @@ func (cr *Compose) GetValue(ctx context.Context, key string, opts ...ropts.Optio
 	return cr.ValueStore.GetValue(ctx, key, opts...)
 }
 
+// SearchValue searches for the value corresponding to given Key.
+func (cr *Compose) SearchValue(ctx context.Context, key string, opts ...ropts.Option) (<-chan []byte, error) {
+	if cr.ValueStore == nil {
+		return nil, routing.ErrNotFound
```

Comment:
I think the appropriate response here may be a closed channel and no error (nothing to find). Thoughts?